rosdep/python.yaml

```diff
@@ -5680,6 +5680,13 @@ python3-mock:
   openembedded: [python3-mock@meta-ros]
   rhel: ['python%{python3_pkgversion}-mock']
   ubuntu: [python3-mock]
+python3-msgpack:
+  debian: [python3-msgpack]
```

Comment:
It looks like this is only available in Debian as of Buster.  Please modify this to:

```
debian:
  '*': [python3-msgpack]
  stretch: null
```